### PR TITLE
Login onboarding

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hey Team,

We've just finished building the LoginOnboard component and page. So feel free to check in any time you are free.  

This is the [PR](https://github.com/onja-org/vamalaka-dong/compare/login_onboarding?expand=1)

Thanks in advance!

### CODE REVIEW

**Review by Ny-Rianala — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function App() {
   return (
     <div className='App'>
       <header className='App-header'>
-        <RoleSelectOptionPage />
+        <LoginOnboardPage />
         {/* <h1>Vamalaka - Madagascars peer-to-peer e-commerce platform</h1>
         <div onClick={() => dispatch(decrement())}>DEC</div>
         <div onClick={() => dispatch(incrementByAmount(10))}>+10</div>
```
Comment (andrzej-onja) [inline src/App.tsx]: remove commented out code

File: src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import './App.css'
-import RoleSelectOptionPage from './pages/RoleSelectOptionPage'
+import LoginOnboardPage from './pages/LoginOnboardPage'
 
 // import adsSelector, decrement, fetchAds, incrementByAmount from'./redux/slices/adsSlice'
 // import { /*useAppSelector,*/ useAppDispatch } from './redux/hooks'
```
Comment (andrzej-onja) [inline src/App.tsx]: remove commented out code

File: src/components/LoginOnboard/LoginOnboard.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  font-family: Futura Std;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-align: right;
+  color: #041d42;
+`
+
+export const LoginOnboard: React.FC = () => {
```
Comment (andrzej-onja) [inline src/components/LoginOnboard/LoginOnboard.tsx]: Lets drop 'onboard' (its more related to registration) just "Login" is fine

File: src/pages/LoginOnboardPage.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import GenericPage from './GenericPage'
+import { LoginOnboard } from '../components/LoginOnboard/LoginOnboard'
+
+const RoleSelectOnboardPage: React.FC = () => {
```
Comment (andrzej-onja) [inline src/pages/LoginOnboardPage.tsx]: rename to LoginPage

File: src/components/Login/Login.tsx [hunk 5]
```diff
@@ -1,44 +1,181 @@
...
+    margin: auto;
+    margin-bottom: -125px;
+  `};
+  ${mediaQueries(null, 'lg')`
+    & > div:nth-child(1) {
+      display: none;
+    }
+  `};
+`
+const LoginOnboardContainer = styled.div`
```
Comment (andrzej-onja) [inline src/components/Login/Login.tsx:34]: just LoginContainer

**Review by andrzej-onja — APPROVED** [review]

**Review by Mickaellah — APPROVED** [review]

**Pull request merged**